phpunit.xml

```diff
@@ -1,40 +1,40 @@
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
->
-    <testsuites>
-        <testsuite name="Test Suite">
-            <directory suffix="Test.php">./tests/</directory>
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./src/</directory>
-
-            <exclude>
-                <directory suffix=".php">./src/*/*/resources</directory>
-                <directory suffix=".php">./src/*/*/*/Legacy</directory>
-            </exclude>
-        </whitelist>
-    </filter>
-    <php>
-        <env name="APP_KEY" value="base64:aYWlZmotmdYnjvN9AAhDXSBl7Vo6RkZCu7eJ9pYJyt8="/>
-        <env name="APP_ENV" value="testing"/>
-        <env name="CACHE_DRIVER" value="array"/>
-        <env name="SESSION_DRIVER" value="array"/>
-        <env name="QUEUE_DRIVER" value="sync"/>
-        <env name="DB_CONNECTION" value="sqlite"/>
-        <env name="DB_DATABASE" value=":memory:"/>
-        <env name="TENANCY_MYSQL_HOST" value="mysql"/>
-        <env name="TENANCY_DB" value="testing"/>
-        <env name="TENANCY_USERNAME" value="mysql_admin"/>
-        <env name="TENANCY_PASSWORD" value="mysql_admin_password"/>
-    </php>
+<phpunit  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+          backupGlobals="false" 
+          backupStaticAttributes="false" 
+          bootstrap="vendor/autoload.php" 
+          colors="true" 
+          convertErrorsToExceptions="true" 
+          convertNoticesToExceptions="true" 
+          convertWarningsToExceptions="true" 
+          processIsolation="false" 
+          stopOnFailure="false" 
+          xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage processUncoveredFiles="true">
+    <include>
+      <directory suffix=".php">./src/</directory>
+    </include>
+    <exclude>
+      <directory suffix=".php">./src/*/*/resources</directory>
+      <directory suffix=".php">./src/*/*/*/Legacy</directory>
+    </exclude>
+  </coverage>
+  <testsuites>
+    <testsuite name="Test Suite">
+      <directory suffix="Test.php">./tests/</directory>
+    </testsuite>
+  </testsuites>
+  <php>
+    <env name="APP_KEY" value="base64:aYWlZmotmdYnjvN9AAhDXSBl7Vo6RkZCu7eJ9pYJyt8="/>
+    <env name="APP_ENV" value="testing"/>
+    <env name="CACHE_DRIVER" value="array"/>
+    <env name="SESSION_DRIVER" value="array"/>
+    <env name="QUEUE_DRIVER" value="sync"/>
+    <env name="DB_CONNECTION" value="sqlite"/>
+    <env name="DB_DATABASE" value=":memory:"/>
+    <env name="TENANCY_MYSQL_HOST" value="mysql"/>
+    <env name="TENANCY_DB" value="testing"/>
+    <env name="TENANCY_USERNAME" value="mysql_admin"/>
+    <env name="TENANCY_PASSWORD" value="mysql_admin_password"/>
+  </php>
 </phpunit>
```

Comment:
What is this? Linting? Update or something else?

Reply:
It's the new config format for phpunit 9 (it was upgraded)